celavi/pylca_celavi/des_interface.py

```diff
@@ -1,304 +1,341 @@
+import os
```

Comment:
@tapajyoti-ghosh there are a lot of changes in this commit. I'll approve but a quick 10 min walkthrough at our next meeting would be good.